tests/vm_userdata_test.go

```diff
@@ -136,9 +137,22 @@ var _ = Describe("CloudInit UserData", func() {
 				})
 
 				LaunchVM(vm)
+
+				By("executing a user-data script")
 				VerifyUserDataVM(vm, []expect.Batcher{
 					&expect.BExp{R: expectedUserData},
 				}, time.Second*120)
+
+				By("applying the hostname from meta-data")
```

Comment:
@karmab I have taken the liberty to add the test for you, since it also required some test-refactoring and was therefore not that trivial. Cirros reflects the hostname in the login line which broke the automated console login:

```
login as 'cirros' user. default password: 'gocubsgo'. use 'sudo' for root.
testvm-nocloud login: 
```